src/charm.py

```diff
@@ -191,6 +191,8 @@ def get_env_vars(self) -> dict:
         """Map config to env vars and return a processed dict."""
         env_vars = utils.map_config_to_env_vars(self)
 
+        env_vars["LIVEPATCH_CONFIG_LOCATION"] = "/etc/livepatch.yaml"
```

Comment:
I don't think we need this, because the charm provides config via environment variables.

Reply:
The server entrypoint (`/cmd/srv/main.go`) expects that env var, otherwise it'll exit.

Reply:
Interesting, do you know why it was working before?

Reply:
Yeah, because the env var was set in the Docker [file]( https://github.com/canonical/livepatch-server/blob/84d212e1f6b2f291324ece044fdbf1a2d65dc40f/docker/Dockerfile#L48). But with the ROCK we don't have that option.